.travis.yml

```diff
@@ -1,7 +1,47 @@
 language: php
+
+sudo: false
+
+cache:
+    directories:
+        - $HOME/.composer/cache
+
 php:
-    - 5.5
     - 5.6
     - 7.0
-install: composer install --no-interaction
-script: ./vendor/bin/phpunit
+    - 7.1
+
+env:
+    global:
+        - MOODLE_BRANCH=MOODLE_32_STABLE
+    matrix:
```

Comment:
This is probably not needed. I'm fairly certain this exists so that the tests are ran against those DBs but since we test against memory to improve build times this isn't required.